setup.py

```diff
@@ -12,22 +11,15 @@
 
 setup(
     name="Products.isurlinportal",
-    version="1.2.2.dev0",
+    version="2.0.0.dev0",
     description="Replacement for isURLInPortal method in Plone",
     long_description=long_description,
     # Get more strings from https://pypi.org/classifiers/
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Framework :: Plone",
-        "Framework :: Plone :: 4.3",
-        "Framework :: Plone :: 5.0",
-        "Framework :: Plone :: 5.1",
-        "Framework :: Plone :: 5.2",
```

Comment:
Should this be kept? And add 6.0? 🤔

Reply:
Oops, did I remove them all?  I meant to replace them all with 6.0 only.  Fixed now.
I don't want to keep 5.2 support.